openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 {% load ace %}
+{% load acetags %}
```

Comment:
Where is this established? I didn't see it in either PR.
And curious, what does the `load ace` above do? Is it not possible to access the tag that way?

Reply:
ace are a set of templatetags inside ace_common djangoapp in edx-platform
i moved the new templatetag to the ace pr so that it would be possible to test with the "full email" per your suggestion, so acetags is the templatetags from ace
probably this could be named something better...though i'm not sure what
i suppose at least i could add a comment in this template

Reply:
Yeah, that could be nice. We don't interact with these templates often enough that it might be good to have reminders of what these load calls are doing. I think the name is fine fwiw.